### DISCUSSION TITLE
Quantification and Depth of Penetration in tDCS

### DISCUSSION THREAD

**Discussion by Croker51o** [simnibs/simnibs#410] [9. Results and analysis] [answered]
Dear Esteemed Experts,

I’m a beginner who has recently started learning about tDCS simulations. I apologize if this question seems basic. I have a few queries that I hope you can help with.

Firstly, I understand that varying the stimulation intensity in tDCS can yield different results. However, I am curious about how to quantify the extent to which stimulation reaches the brain sulcus depending on the intensity.

Secondly, I would like to know whether this process can be achieved using either a GUI or Python. I am interested in the specific steps involved in carrying out these operations.

I would greatly appreciate any insights or advice from those of you with more experience in this field.

Sincerely,
Cr

**Comment by diedrim** [comment, marked as answer]
Please have a look at these tutorials: https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html, https://simnibs.github.io/simnibs/build/html/tutorial/analysis.html.
If you have further questions, please reach out.

**Reply by Croker51o** [reply]
Thank you so much for your response. I’ll start by trying out the advice you gave me. I really appreciate your help.
